src/org/ohdsi/usagi/ui/actions/UsagiDialogs.java

```diff
@@ -0,0 +1,54 @@
+package org.ohdsi.usagi.ui.actions;
+
+import org.ohdsi.usagi.ui.Global;
+
+import javax.swing.*;
+
+public class UsagiDialogs {
```

Comment:
This is a good start at separating the UI from the logic. It could be slightly improved by making this a service instead of a static method class, and perhaps by allowing custom dialogs to be started.